sdk/servicebus/azure-servicebus/swagger/README.md

```diff
@@ -17,7 +17,7 @@ autorest --use=@autorest/python@5.0.0-preview.6
 ```
 ### Settings
 ``` yaml
-input-file: servicebus-swagger.json
+input-file: C:/Work/azure-rest-api-specs-pr/specification/servicebus/data-plane/servicebus-swagger.json
```

Comment:
Weren't we moving to an HTTP based consumption of this to prevent a dev needing to pull this down?

Reply:
HTTP based requires token. Let me ask how Storage avoid token first.

Reply:
I pasted the url with a token.